tools/spi-hdlc-adapter/spi-hdlc-adapter.c

```diff
@@ -102,29 +103,33 @@ static int gMode = MODE_PTY;
 static int gMode = MODE_STDIO;
 #endif
 
-static const char* gSpiDevPath = NULL;
+static const char* gSpiDevPath     = NULL;
```

Comment:
`<soapbox>`Was never really a fan of using an `s` prefix, probably due to my association of the prefix with the `sz`/`st` prefixes from [hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation).`</soapbox>`

Will fix.